spec/integration/bulk_jobs_spec.rb

```diff
@@ -17,7 +17,7 @@
     FileUtils.rm_rf(@output_directory) if(Dir.exist?(@output_directory))
   end
 
-  describe 'perform' do
+  describe 'perform', :integration => true do
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.